docs/src/DatePicker.doc.js

```diff
@@ -0,0 +1,398 @@
+  'nb-NO': { localeData: nb, lang: ' Norwegian (Bokm\u00e5l)' },
+  nl: { localeData: nl, lang: ' Dutch' },
+  'pl-PL': { localeData: pl, lang: ' Polish (Poland)' },
+  'pt-BR': { localeData: ptBR, lang: ' Portuguese (Brazilian)' },
+  'pt-PT': { localeData: pt, lang: ' Portuguese (Portugal)' },
+  'ro-RO': { localeData: ro, lang: ' Romanian' },
+  'ru-RU': { localeData: ru, lang: ' Russian' },
+  'sk-SK': { localeData: sk, lang: ' Slovak' },
+  'sv-SE': { localeData: sv, lang: ' Swedish' },
+  'th-TH': { localeData: th, lang: ' Thai' },
+  tr: { localeData: tr, lang: ' Turkish' },
+  'uk-UA': { localeData: uk, lang: ' Ukrainian' },
+  'vi-VN': { localeData: vi, lang: ' Vietnamese' },
+  'zh-CN': { localeData: zhCN, lang: ' Chinese (Simplified)' },
+  'zh-TW': { localeData: zhTW, lang: ' Chinese (Traditional)' },
+};
+
+card(
+  <PageHeader
+    name="DatePicker"
+    description="Use Datepicker when the user has to select a date or date range."
+    beta
+  />
+);
+
+card(
+  <PropTable
+    props={[
+      {
+        name: 'accessibilityLabelIcon',
+        required: true,
+        type: 'string',
+        description:
+          'Accessibility label for the calendar icon on the date field input that clients such as VoiceOver will read to describe the element.',
+      },
+      {
+        name: 'accessibilityLabelNextMonth',
+        required: true,
+        type: 'string',
+        description:
+          'Accessibility label for the next-month button on the calendar view that clients such as VoiceOver will read to describe the element.',
+      },
+      {
+        name: 'accessibilityLabelPreviousMonth',
+        required: true,
+        type: 'string',
+        description:
+          'Accessibility label for the previous-month button on the calendar view that clients such as VoiceOver will read to describe the element.',
+      },
+      {
+        name: 'helperText',
+        type: 'Node',
+        description:
+          'More information about how to complete the date picker field',
+        href: 'helperText',
+      },
+      { name: 'disabled', type: 'boolean', href: 'disabled' },
+      {
+        name: 'endDate',
+        type: 'Date',
+        description:
+          'End date for which selection is enabled. Required for date range picker.',
+        href: 'rangePicker',
+      },
+      { name: 'errorMessage', type: 'Node', href: 'errorMessage' },
+      {
+        name: 'excludeDates',
+        type: 'Array<Date>',
+        description: 'Array of disabled dates for selection',
+      },
+      { name: 'id', required: true, type: 'string' },
+      {
+        name: 'idealDirection',
+        type: `'up'|'right'|'down'|'left'`,
+        description: 'Preferred direction for the calendar flyout to open',
+      },
+      {
+        name: 'includeDates',
+        type: 'Array<Date>',
+        description: 'Array of enabled dates for selection',
+      },
+      {
+        name: 'isRTL',
```

Comment:
@AlbertCarreras can we change this to `layoutDirection: 'rtl' | 'ltr'`?

Reply:
reminder changer arrow css

Reply:
fixed